core/integration/features/verified/closeout-2 partyies_lognormal.feature

```diff
@@ -0,0 +1,130 @@
+      | party     | market id | side | volume | price  | resulting trades| type       | tif     | reference |
+      | auxiliary2| ETH/DEC19 | buy  | 5      | 5      | 0               | TYPE_LIMIT | TIF_GTC | aux-b-5   |
+      | auxiliary1| ETH/DEC19 | sell | 10     | 1000   | 0               | TYPE_LIMIT | TIF_GTC | aux-s-1000|
+      | auxiliary2| ETH/DEC19 | buy  | 10     | 10     | 0               | TYPE_LIMIT | TIF_GTC | aux-b-1   |
+      | auxiliary1| ETH/DEC19 | sell | 10     | 10     | 0               | TYPE_LIMIT | TIF_GTC | aux-s-1   |
+    When the opening auction period ends for market "ETH/DEC19"
+    And the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "ETH/DEC19"
+    Then the auction ends with a traded volume of "10" at a price of "10"
+    And the mark price should be "10" for the market "ETH/DEC19"
+
+    # setup trader2 position to be ready to takeover trader3's position once trader3 is closed out
+    When the parties place the following orders:
+      | party     | market id | side | volume| price | resulting trades | type       | tif     | reference   |
+      | trader2   | ETH/DEC19 | buy  | 40    | 50    | 0                | TYPE_LIMIT | TIF_GTC | buy-order-3 |
+    Then the order book should have the following volumes for market "ETH/DEC19":
+      | side | price  | volume   |
+      | buy  | 5      | 5        |
+      | buy  | 50     | 4040     | 
+      | sell | 1000   | 10       | 
+      | sell | 1055   | 223      | 
+
+    And the parties should have the following margin levels:
+      | party     | market id  | maintenance | search | initial | release |
+      | trader2   | ETH/DEC19  | 321         | 481    | 642     | 963     |
+      # margin level = OrderSize*MarkPrice*RF = 40*10*0.801225765=321
+
+    Then the parties should have the following account balances:
+      | party   | asset | market id | margin   | general |
+      | trader2 | USD   | ETH/DEC19 | 642      | 1358    |
+
+    # setup trader3 position and close it out
+    When the parties place the following orders:
+      | party      | market id | side | volume| price | resulting trades | type       | tif     | reference      |
+      | trader3    | ETH/DEC19 | buy  | 10    | 100   | 0                | TYPE_LIMIT | TIF_GTC | buy-position-31 |
+      
+    Then the order book should have the following volumes for market "ETH/DEC19":
+      | side | price  | volume   |
+      | buy  | 5      | 5        |
+      | buy  | 45     | 278966   |
+      | buy  | 50     | 40       | 
+      | buy  | 100    | 10       | 
+      | sell | 1000   | 10       | 
+      | sell | 1055   | 223      | 
+
+    And the parties should have the following margin levels:
+      | party     | market id  | maintenance | search | initial | release |
+      | trader3   | ETH/DEC19  | 81          | 121    | 162     | 243     |
+
+    # how come trader3's order is not rejected? they only have 90 in the collateral and the initial margin level is 162
```

Comment:
Isn't this the issue recently discussed on Slack where a spec change was proposed? I think the comment should be rephrased to explain why this is allowed.